sdk/tests/tutorials/properties/test_percision.py

```diff
@@ -63,7 +62,7 @@ def test_precision(self):
 
 
         self.assertTrue(type(portfolio.properties['Portfolio/test-precision/decimal_value'].value.metric_value.value)== Decimal)
-        self.assertEqual(Decimal('0.12345678910111213'), portfolio.properties['Portfolio/test-precision/decimal_value'].value.metric_value.value)
+        self.assertEqual(Decimal('0.1234567891011121298916464184'), portfolio.properties['Portfolio/test-precision/decimal_value'].value.metric_value.value)
```

Comment:
How come this does not equal `0.1234567891011121314151617182`?

Reply:
I was missing some qoutes, this is fixed now